datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: HOME_PAGE_DOMAINS_ID,
+        selector: `#${HOME_PAGE_DOMAINS_ID}`,
+        title: 'Explore by Domain',
+        content: (
+            <Typography.Paragraph>
+                Here are your organization&apos;s <strong>Domains</strong>. Domains are collections of data assets -
+                such as Tables, Dashboards, and ML Models - that make it easy to discover information relevant to a
+                particular part of your organization.
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: HOME_PAGE_PLATFORMS_ID,
+        selector: `#${HOME_PAGE_PLATFORMS_ID}`,
+        title: 'Explore by Platform',
+        content: (
+            <Typography.Paragraph>
+                Here are your organization&apos;s <strong>Data Platforms</strong>. Data Platforms represent specific
+                third-party Data Systems or Tools. Examples include Data Warehouses like <strong>Snowflake</strong>,
+                Orchestrators like
+                <strong>Airflow</strong>, and Dashboarding tools like <strong>Looker</strong>.
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: HOME_PAGE_MOST_POPULAR_ID,
+        selector: `#${HOME_PAGE_MOST_POPULAR_ID}`,
+        title: 'Explore Most Popular',
+        content: "Here you'll find the assets that are viewed most frequently within your organization.",
+    },
+    {
+        id: HOME_PAGE_SEARCH_BAR_ID,
+        selector: `#${HOME_PAGE_SEARCH_BAR_ID}`,
+        title: 'Find your Data 🔍',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    This is the <strong>Search Bar</strong>. It will serve as your launch point for discovering and
+                    collaborating around the data most important to you.
+                </p>
+                <p>
+                    Not sure where to start? Click on <strong>Explore All</strong>!
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: SEARCH_RESULTS_FILTERS_ID,
+        selector: `#${SEARCH_RESULTS_FILTERS_ID}`,
+        title: 'Narrow your search ⚡',
+        content: (
+            <Typography.Paragraph>
+                Quickly find relevant assets by applying one or more filters. Try filtering by{' '}
+                <strong>Entity Type</strong>, <strong>Owner</strong>, and more!
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: SEARCH_RESULTS_ADVANCED_SEARCH_ID,
+        selector: `#${SEARCH_RESULTS_ADVANCED_SEARCH_ID}`,
+        title: 'Dive Deeper with Advanced Search 💪',
+        content: (
+            <Typography.Paragraph>
+                Use <strong>Advanced Search</strong> to find specific assets using granular filter predicates.
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_ENTITIES_ID,
+        selector: `[id^='rc-tabs'][id$='Entites']`,
```

Comment:
ah yeah so I bet this would change if we rename the tab (no action needed, just calling it out)

Reply:
+1

Reply:
Added a comment for future reference